optuna_dashboard/ts/action.ts

```diff
@@ -265,6 +285,44 @@ export const actionCreator = () => {
       })
   }
 
+  const tellTrial = (
+    studyId: number,
+    trialId: number,
+    state: TrialState,
+    values?: string[]
```

Comment:
To increase the reusability, could you parse values before this function calls? For example, if we add the feature to specify objective values with a slider or radio button, this action would be called without parsing the string.

```suggestion
    values?: number[]
```

Reply:
Fixed.